roles/storage-cns/README.md

```diff
@@ -11,3 +11,6 @@ installed.
 |admin_password| |_optional_|Password for **admin_user**.|
 |cluster |openshift |<ul><li>openshift</li><li>kubernetes</li></ul>|Cluster type.| 
 |action |provision| <ul><li>provision</li><li>deprovision</li></ul>|Action to perform.|
+| heketi_route_name | heketi-storage | | The Openshift route that exposes the heketi service |
+| heketi_namespace | app-storage | | The namespace where heketi is deployed |
+| heketi_secret_name | heketi-storage-admin-secret | | The name of the secret containing heketi credentials |
```

Comment:
14,16s/heketi/Heketi
and description should be in full sentances, i.e. with full stops.